app/static/js/events/wizard/vue-app.js

```diff
@@ -1,4 +1,6 @@
 var steps = ['', 'sponsors', 'sessions-tracks-rooms'];
+var modifications = {'sessions-tracks-rooms': {sessionCreate:0, sessionRemove:0, trackCreate:0, trackRemove:0, roomCreate:0, roomRemove:0},
```

Comment:
@niranjan94 That's a good idea. I am making a pull request giving the co-organizer the same permissions as the organizer for the wizard permissions. Closing this request for a while.